### ISSUE TITLE
Material Editor: In Pick Document window Delete option is missing from files context menu

### ISSUE THREAD

**Issue by LB-BartoszCwiek** [o3de/o3de#10854]
**Describe the bug**
In the Pick Document window, the files context menu lacks Delete option, which is functional when using Del keybind.

**Steps to reproduce**
1. Go to Tools > Material Editor.
2. Go to File > Open.
3. Search for any .material file.
4. RMB on the searched file.

**Expected behavior**
Delete option is missing from the files context menu.

**Actual behavior**
Delete option is available from the files context menu and from its shortcut.

**Video**

https://user-images.githubusercontent.com/100559077/180774697-84fad9fb-ee22-4110-bf33-4cbbc250a6ad.mp4

**Screenshot**

![MaterialEditorImage](https://user-images.githubusercontent.com/100559077/180774777-a77c5b45-a5f7-4538-ae52-2bc0eab2a04c.png)

**Found in Branch**
Development (13bd32d)

**Desktop:**
- Device: PC
- OS: Windows
- Version: 10
- CPU: Intel Core i7-11700F
- GPU: NVIDIA GeForce RTX 3070

**Comment by gadams3** [comment]
I don't believe it was ever intended to support deleting files while attempting to open them. Closing this issue as designed.
